src/accounts/contacts/Dockerfile

```diff
@@ -37,4 +37,7 @@ ENV LOG_LEVEL info
 # Add application code.
 COPY . .
 
+# Ensure files are readable for non-root users
+RUN chmod a+r *
```

Comment:
This defeats the purpose of running in non-root, since it gives gives access to **all** files to non-root, potentially creating security issues.

Reply:
Only to the application code and config files in `/app` -- I can make this more explicit if helpful. Another option would be to chown these to the user/group the pod is configured to run as.